Codeinwp/neve-fse
# feat: add survey notice #32

### Summary
Added survey notice to dashboard and themes page.
The notice displays after 3 days.

**Note:** To display this I replicated the option that we set via the SDK as for the moment this is not part of the theme.

### Will affect visual aspect of the product

YES

### Screenshots 

![image](https://github.com/Codeinwp/neve-fse/assets/23024731/5832799a-e68d-40ba-b496-3ca272a1ff2d)


### Test instructions

1. Use the build from this PR.
2. Activate the Neve FSE theme.
3. Check that the notice is not present.
4. Run `wp option update neve_fse_install 1685187068` or any timestamp in the past (3 days earlier)
5. Check that the notice is displayed.
6. Check the link to the survey.
7. Check that the notice is dismissable.
8. If 'Maybe later' is clicked it will dismiss the notice for the current display, will appear on refresh.
9. If it is closed using the regular dismiss it will dismiss it forever.


Closes #32.
